tests/FakeJob.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Tests;
+
+use Illuminate\Contracts\Queue\Job as JobContract;
+use Illuminate\Queue\Jobs\FakeJob as JobsFakeJob;
+
+class FakeJob extends JobsFakeJob implements JobContract
+{
+    //
+}
```

Comment:
The framework's fake job does not implement the job contract. All other job types do and it is expected that 3rd party jobs also implement the contract.

I've opened a PR to add this to the framework. We will need to keep this for a while.